packages/management/addons-manager/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: addons-manager.tanzu.vmware.com.1.5.0
+  namespace: addons-manager
+spec:
+  refName: addons-manager.tanzu.vmware.com
+  version: 1.5.0
+  releaseNotes: "addons-manager 1.5.0"
+  licenses:
+    - "UNKNOWN"
```

Comment:
Could we use ```Copyright 2022 VMware, Inc. All Rights Reserved.
SPDX-License-Identifier: Apache-2.0``` which is the same as header in all go files?

Reply:
@vijaykatam only in this yaml file or all the yamls we have for addons-manager?

Reply:
@vijaykatam I followed the other 2 package.yaml files in management package so didn't add this header, let me know if we have to add it.